lib/ask/runtime/channel_broker.ex

```diff
@@ -36,8 +36,8 @@ defmodule Ask.Runtime.ChannelBroker do
     call(channel_id, {:has_delivery_confirmation?})
   end
 
-  def ask(channel_id, channel_type, respondent, token, reply) do
-    cast(channel_id, {:ask, channel_type, respondent, token, reply})
+  def ask(channel_id, channel_type, respondent, token, reply, not_before \\ nil, not_after \\ nil) do
```

Comment:
Tempted to refactor so that nil's aren't accepted, but I wonder if there are other parts of Surveda where a schedule is really optional

Reply:
Yeah, the simulator came to mind. I can do it then. Let's pray that test coverage is enough :D

Reply:
Mmm, I just noticed `ask` is also used for mobile web mode, and in that one we don't care about schedules